ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/fullyprivate/cco-manual-workload-identity/provision/cucushift-installer-rehearse-azure-ipi-fullyprivate-cco-manual-workload-identity-provision-chain.yaml

```diff
@@ -0,0 +1,30 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-fullyprivate-cco-manual-workload-identity-provision
+  steps:
+  - ref: azure-provision-resourcegroup
+  - ref: azure-provision-vnet
+  - ref: azure-provision-nat
+  - chain: azure-provision-bastionhost
+  - ref: proxy-config-generate
+  - chain: ipi-conf-azure
+  - ref: ipi-conf-azure-provisionednetwork
+  - ref: ipi-conf-azure-basedomainresourcegroup
```

Comment:
we have a dedicated step to do the setting in `ipi-conf-azure`, which is included in `chain: ipi-conf-azure`, do we really need this one, sounds it is duplicated.